src/frontend/src/tables/InvenTreeTable.tsx

```diff
@@ -501,6 +508,30 @@ export function InvenTreeTable<T = any>({
     });
   }, [tableState.selectedRecords]);
 
+  // Callback when a row is clicked
+  const handleRowClick = useCallback(
+    (record: any, index: number, event: any) => {
+      if (props.onRowClick) {
+        // If a custom row click handler is provided, use that
+        props.onRowClick(record, index, event);
+      } else if (tableProps.modelType && record?.pk) {
+        // If a model type is provided, navigate to the detail view for that model
+        let url = getDetailUrl(tableProps.modelType, record.pk);
+
+        // Should it be opened in a new tab?
+        if (event?.ctrlKey || event?.shiftKey || event?.buttons & 0x04) {
+          // Open in a new tab
+          // TODO: Remove /platform/ prefix later on
+          window.open('/platform' + url, '_blank', 'noreferrer');
```

Comment:
I think we have a variable for the relative path somewhere we could use here, just import it from `main.tsx`:

https://github.com/inventree/InvenTree/blob/08c0953eb73741d1754fd9cc8169793e553a00d2/src/frontend/src/main.tsx#L82

Reply:
Oh nice, thanks!